src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -72,6 +72,7 @@ export type DropdownMenuProps<T> = {
      * Overrides the default dropdown popup props.
      */
     popupProps?: Partial<PopupProps>;
+    rowActionsSize?: ButtonSize;
```

Comment:
When I said DropdownMenu, I didn't mean the actual component. I mean Popup with Menu inside withTableActions file. Please revert changes in DropdownMenu